README.md

```diff
@@ -1,2 +1,103 @@
-# Chris-and-Ed-Project3-API-calls
-Node.js and express API project implementing image vision description and TTS
```

Comment:
yep, will add maybe webscraping and image search also! but can add that myself in a revision